Kiosk/Bid Fulfillment/ManualCreditCardInputViewController.swift

```diff
@@ -36,34 +36,34 @@ class ManualCreditCardInputViewController: UIViewController, RegistrationSubCont
         billingZipWrapperView.isHidden = true
 
         // We show the enter credit card number, then the date switching the views around
-        viewModel
-            .cardFullDigits
+        cardNumberTextField.rx.text
             .asObservable()
-            .bind(to: cardNumberTextField.rx.text)
+            .replaceNilWith("")
```

Comment:
`cardNumberTextField.rx.text` produces a string of `Optional<String>`, and `replaceNil("")` will replace the optional values with the empty string.